14-truck_backer-upper.ipynb

```diff
@@ -92,12 +119,12 @@
     "        \n",
     "        # Check for illegal conditions\n",
     "        if self.is_jackknifed():\n",
-    "            print('The truck is jackknifed!')\n",
-    "            return\n",
+    "            # print('The truck is jackknifed!')\n",
```

Comment:
This can be uncommented if you like, It was just removed so it is not constantly printed during training. Same with the other comment on the car being off screen you posted.